gradle.properties

```diff
@@ -1,4 +1,4 @@
 // Project properties
-version=2.4.0
+version=2.5.0
```

Comment:
I don't think we need to go 2.5.0 for this. I would combine it in 2.4.0 Please update if possible. If not, I can take care of this.

Reply:
Sorry, I had not noticed 2.4.0 had not been released yet. That's done.